_posts/2018-10-23-gsoc-setup-assistant-v2.markdown

```diff
@@ -0,0 +1,45 @@
+---
+author: nbbrooks
+comments: false
+date: 2018-10-23 15:50:00+00:00
+layout: post
+slug: gsoc-2018-setup-assistant-v2
+title: Google Summer of Code Improvements to Setup Assistant
+media_type: image
+media_link: http://moveit.ros.org/assets/images/gsoc.png
+description: Summary of Google Summer of Code improvements to the MoveIt! Setup Assistant.
+categories:
+- MoveIt!
+- ROS
+---
+
+Submitted by: Mohamad Ayman, Cairo University
+
+For this year's Google Summer of Code Project (GSoC) 2018, with coordination with PickNik Consulting, Open Source Robotics Foundation (OSRF), and Dave Coleman, and Andy McEvoy, Setup Assistant v2 has been developed.
+
+The focus of this research was to improve the Setup Assistant.  The idea was to add more features to the Moveit! Setup Assistant as well as to simplify setting up robots for simulations and other tasks. This blog post will further explain how to simulate your Moveit! Controlled robot in Gazebo using the upgraded setup assistant and act as a tutorial
```

Comment:
```suggestion
The focus of this research was to improve the Setup Assistant.  The idea was to add more features to the Moveit! Setup Assistant as well as to simplify setting up robots for simulations and other tasks. This blog post will further explain how to simulate your MoveIt! controlled robot in Gazebo using the upgraded Setup Assistant and act as a tutorial.
```

Reply:
How is this acting as a tutorial?

Reply:
I think the "Overview" section is being referred to here as a [mini] tutorial.